src/Request/RequestFactoryInterface.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Psr\Http\Factory;
+namespace Psr\Http\Factory\Request;
```

Comment:
I like the grouping for it is easy to look less files. Eg : When `Request` and `Response` are on same directory it is hard. Hard time to read the 2nd one. @mindplay-dk  the first one seems much better in that case.